routes/api.js

```diff
@@ -379,6 +379,58 @@ router.post("/admin/addHomeVid", (req, res) => {
   } else res.status(403).send("403 - Forbidden. You are not authorized to make requests here.");
 });
 
+router.post("/admin/addModule/:id", (req, res) => {
```

Comment:
This is fine for now but its not best practice to include the id in the url. I know we did it for one of the other module ones